Rules.md

```diff
@@ -223,9 +224,9 @@ A battle has one or more rounds. Every round has following parts:
       another region under attack. Healthy retreated units are allowed to participate in the
       following battle.
 
-  The attacker moves all attacking units into the region. The invested coins in the conquered region
-  may be kept on the board or may be (partly) stolen. If a token is present, the attacker may use it
-  in a next command round after which the token returns to the defender.
+  The attacker moves all attacking units into the region and may plunder the region. If a token is
```

Comment:
I see, but I would not introduce a new concept (plunder). I'd prefer to repeat the rule here.